src/main/java/net/satago/tapestry5/jpa/TransactionalUnit.java

```diff
@@ -33,7 +33,7 @@
 
```

Comment:
Yes, I like the idea of popping them off the stack once done. A transactional unit before commit is an interesting edge but there's no pressure to resolve all of it right now. Related to your earlier comment about T5 version, it probably makes the most sense to offer this module for pre T5.5 versions. It probably requires some amount of duplicate effort on merging any fixes but likely not a major issue.